diode-server/docker/sample.env

```diff
@@ -5,7 +5,7 @@ REDIS_HOST=diode-redis
 REDIS_PORT=6378
 RECONCILER_GRPC_HOST=diode-reconciler
 RECONCILER_GRPC_PORT=8081
-NETBOX_DIODE_PLUGIN_API_BASE_URL=http://NETBOX_HOST/api/plugins/diode
+NETBOX_DIODE_PLUGIN_API_BASE_URL=http://192.168.31.114:8000/netbox/api/plugins/diode
```

Comment:
Can we revert this change, please? It's a local variable IP

Reply:
```suggestion
NETBOX_DIODE_PLUGIN_API_BASE_URL=http://NETBOX_HOST/api/plugins/diode
```